tensorflow/compiler/xla/backends/profiler/gpu/device_tracer_rocm.cc

```diff
@@ -181,16 +181,9 @@ class RocmTraceCollectorImpl : public profiler::RocmTraceCollector {
 
   void Export(XSpace* space) {
     uint64_t end_gputime_ns = RocmTracer::GetTimestamp();
-<<<<<<< HEAD:tensorflow/core/profiler/backends/gpu/device_tracer_rocm.cc
```

Comment:
I don't think this is correct.

The other part of the conflict should be used:

```
    XPlaneBuilder host_plane(
        FindOrAddMutablePlaneWithName(space, kRoctracerApiPlaneName));
    int i = 0;
    for (auto& device : per_device_collector_) {
```

Reply:
Yes, please check my commit on fixed build error. Thanks